packages/mobile/lib/auxiliary/drawer/enums/drawer-id.enum.ts

```diff
@@ -0,0 +1,14 @@
+export enum DrawerId {
+    AccountActions = 'account-actions',
+    AccountSwitcher = 'account-switcher',
+    SelectedActivity = 'selected-activity',
+    Profile = 'profile',
+    Send = 'send',
+    Receive = 'receive',
+    SelectedToken = 'selected-token',
+    Legal = 'legal',
+    EnterPassword = 'enter-password',
+    AddReferences = 'add-references',
+    AddExpiration = 'add-expiration',
+    Confirm = 'confirm',
```

Comment:
Do these have to be `kebab-case`? The usual convention for enums is `PascalCase` for the key then `camelCase` for the value.